example5.ts

```diff
@@ -0,0 +1,80 @@
+import * as readline from 'readline';
+
+type Position = {
+  x: number;
+  y: number;
+};
+
+const maze: string[][] = [
```

Comment:
Consider using a more descriptive name than 'maze' for a variable that holds a 2D array. It could be more intuitive for other developers. The Force is strong with this variable.